joomla/joomla-cms
# [4.0] Namespace com_languages

Pull Request for Issue # .

### Summary of Changes
This simple PR converts com_languages component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_languages.zip )

2. Login to administrator area, access to Extensions-> Languages, check and make sure all the features (Installed. Content Languages, Overrides) works as expected (works in the same way as none namespace version - before this PR)
